custom/icds_reports/templates/icds_reports/icds_app/poshan-progress-dashboard.directive.html

```diff
@@ -29,7 +29,9 @@ <h3 class="black">No Data</h3>
                 <tr ng-repeat="indicators in section.indicators">
                   <td ng-repeat="indicator in indicators" class="d-flex justify-content-between col-xs-6 px-0">
                     <p ng-bind="indicator.name" class="col-xs-8 no-margin"></p>
-                    <p ng-bind="indicator.value" class="col-xs-4 text-center no-margin"></p>
+                    <p class="col-xs-4 text-center no-margin">
+                      {$ ($ctrl.notLaunched) ? 'Not Launched' : indicator.value $}
```

Comment:
I think you will need to change this when you will change the above.

Reply:
Got it @Rohit25negi. I left the case of value being null because 
https://github.com/dimagi/commcare-hq/blob/a18267b85ec2bdc403e0d2e68e0567a511bf354a/custom/icds_reports/migrations/sql_templates/database_views/poshan_progress_report_view.sql#L13
here `num_launched_awcs` are stored as 0 in case of null. I thought a case of null would never occur